docs/TASKS.md

```diff
@@ -0,0 +1,26 @@
+# Tasks
+
+With very few exceptions, any running process on EVE has to be part of the [Task](https://github.com/lf-edge/eve/blob/master/pkg/pillar/types/domainmgrtypes.go#L148). EVE's tasks map to containerd [Tasks](https://containerd.io/docs/getting-started/#creating-a-running-task) and ultimately to a group of processes running together in a container. Thus, each task is ultimately defined by an [OCI Runtime spec](https://github.com/opencontainers/runtime-spec/blob/master/spec.md) which can be inspected by the `ctr --namespace <name space name> container info <task name>` command. EVE uses two namespaces for its tasks:
```

Comment:
A reference to a line in domainmgrtypes.go isn't likely to be stable. Don't know if there are other types of references available in github. FWIW for API references the docsmerely reference the file and the reference tag is the name of the type.

Reply:
Good point. We do have: https://godoc.org/github.com/lf-edge/eve/pkg/pillar and this kind of a link should be much more stable https://godoc.org/github.com/lf-edge/eve/pkg/pillar/types#Task

I think I'll replace it with the above link unless you have some better suggestions @eriknordmark

Reply:
Done